### ISSUE TITLE
International names bug

### ISSUE THREAD

**Issue by DL6ER** [Langenscheiss/bibitnow#9]

**Comment by Langenscheiss** [comment]
Thanks for reporting. 

Two things:
1.) I will have to write an adjuster for this harvard page (or you may want to give it a try yourself, it's not very difficult :) ). You see that the "citation_authors" field misses a lot of authors, whereas the "dc.creator" field has all authors. With a few lines of code, one can tell the system to prefer dc.creator, using the "prefselector".
2.) The command \i does exist

https://tex.stackexchange.com/questions/385242/what-is-the-character-i

It means "dotless i", but depending on what version of latex you use, it might not be recognized. If there is a more general scheme behind this (which I don't know yet), this can probably be fixed by another one of those famous "bibtex options", but in the meantime, I suggest you check first whether you can make your system compatible to the command \i. I will investigate this issue.

**Comment by DL6ER** [comment]
> but depending on what version of latex you use, it might not be recognized

This seems to be the case. I use the most recent version of `pdftex` (TeXLive 2018) with `biblatex`+`biber`.

When searching for this online, I often find the suggestion to "just use the UTF-8 character". Could there be an option in BibItNow to don't replace the international characters and just use the UTF-8 variant directly? I just tested this in my setup and using, e.g. `î` directly in my bib file works just fine in my setup.

**Comment by Langenscheiss** [comment]
Yes, I was thinking about this option for a longer time. I am not a fan of UTF-8 in latex, but it seems like this where things are going, so I am going to have to keep up with that. As a quick and dirty fix, I can just add an option to use the strings that are being used in the RIS/Endnote filter. This might cause issues here and there, but it would probably work in most cases for people using fully utf8 ready latex distributions.

**Comment by Langenscheiss** [comment]
In the next version, you get an option to replace \.{\i} and \.{\j} by \.{i} and \.{j} (with the "." being any relevant character). Also solved the Harvard issue (which was actually a problem in my current developer build). Implementing a UTF8 mode is going to take a bit more time and thinking, due to some special characters that nevertheless need to be replaced.

**Comment by Langenscheiss** [comment]
In 0.840 (online now for Firefox), you should have the option in Bibtex (and Barnold S.). The default is "disabled" though, so you have to turn it on

**Comment by DL6ER** [comment]
Confirmed this working as expected. Thanks!

**Comment by Langenscheiss** [comment]
Btw.: this is most likely a biber bug:

https://tex.stackexchange.com/questions/23817/biber-writes-incorrect-unicode-encoding-of-i

This issue is precisely the kind of detail issue that convinced me to write this plugin. There are tons of library systems out there that can extract bibtex, but most of them don't get these details right, or they don't work with the user's preferred tex configuration. And it is exactly these details that unnecessarily consume time -- as in your case. So please, report these issues if you encounter them, it's what BibItNow! is intended to be good at.
